capirca/aclgen.py

```diff
@@ -463,6 +463,19 @@ def _WriteFile(output_file, file_string):
     raise
 
 
+def DiscoverAllPolicies(base_directory, output_directory, definitions):
+  logging.info('finding policies...')
+  pols = []
+  pols.extend(
```

Comment:
Nitty nit: I think it's slightly more pythonic to write pols = [].extend(... ? But maybe I"m wrong?

Reply:
`extend` is a mutating function so it returns None, which means we can't use it in that way